tests/Doctrine/DBAL/Migrations/Tests/Tools/Console/Helper/ConfigurationHelperTest.php

```diff
@@ -73,13 +73,16 @@ protected function getConfigurationHelperLoadsASpecificFormat($baseFile, $config
             $configurationHelper = new ConfigurationHelper($this->getSqliteConnection());
             $configfileLoaded    = $configurationHelper->getMigrationConfig($this->input, $this->getOutputWriter());
 
+            self::assertAttributeSame($this->getOutputWriter(), 'outputWriter', $configfileLoaded);
+
             unlink($configFile);
 
             return trim($this->getOutputStreamContent($this->output));
         } catch (\Exception $e) {
```

Comment:
This should be a `finally`

Reply:
I didn't change to `finally` because this needs to be backported and I wasn't sure about the version, now we saw that we're requiring `^7.1` for `v1.6.x` so it's all good.